share/completions/connmanctl.fish

```diff
@@ -23,27 +23,27 @@ complete -f -c connmanctl -n "test (count (commandline -opc)) -lt 2" -a state -d
 complete -f -c connmanctl -n "test (count (commandline -opc)) -lt 2" -a technologies -d "Display technologies"
 complete -f -c connmanctl -n "test (count (commandline -opc)) -lt 2" -a clock -d "Get System Clock Properties"
 complete -f -c connmanctl -n "test (count (commandline -opc)) -lt 2" -a enable -d "Enables given technology or offline mode"
-complete -f -c connmanctl -n "test (count (commandline -opc)) -eq 2; and contains -- (commandline -opc)[2] enable" -a "(__fish_print_connman_technologies) offline"
+complete -f -c connmanctl -n "fish_is_nth_token 2; and contains -- (commandline -opc)[2] enable" -a "(__fish_print_connman_technologies) offline"
```

Comment:
This decreases local consistency, so I don't think it's worth it.
I'll apply the conda change.